lib/app.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+const net = require('net');
+const ChatRoom = require('./ChatRoom');
+const parseMessage = require('./parseMessage');
+
+const clients = new ChatRoom();
+
+const app = net.createServer((client) => {
+  client.setEncoding('utf8');
+  clients.add(client);  
+  console.log('CLIENT CONNECTED');
+
+  client.on('error', (error) => {
+    console.log(error);
+  });
+
+  client.on('data', data => {
+    console.log(`CLIENT: ${data}`);
+    const { command, arg, text } = parseMessage(data);
```

Comment:
```suggestion
    const message = parseMessage(data);
    if(!message) return;
    const { command, arg, text } = message;
```
Encountered this error when I attempted to have user2 message user1. Try adding a guard clause like above to prevent this.